assets/Components/TouchableButton.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { TouchableHighlight, View } from "react-native";
+import { ButtonText } from './ButtonText'
+
+export default function TouchableButton({onPress, text}) {
+  return (
+    <View>
+      <TouchableHighlight
+        onPress={onPress}
+      >
+        <ButtonText>{text}</ButtonText>
+      </TouchableHighlight>
+    </View>
+
+  )
+}
```

Comment:
Perfect way of building a reusable component. Although, try to stick to the code guide lines we're used to.
```
export const TouchableButton = () => (
  <View>...</View>
)
```